website/yw_website/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
+}
```

Comment:
End files with a [newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
This shouldn't be committed anyway. But a good point nonetheless